spec/index.html

```diff
@@ -8,6 +8,19 @@
 copyright: proposal
 contributors: Dean Tribble, Till Schneidereit, Sathya Gunasekaran
 </pre>
+<style>
+[normative-optional] {
```

Comment:
Could you send me a screenshot of how the normal optional text below gets rendered with these stylings?

Reply:
![image](https://user-images.githubusercontent.com/189835/58912208-d771c380-8719-11e9-9eca-eecdfbe04818.png)

Reply:
Excellent! I am all in favor. Thanks.